src/index.ts

```diff
@@ -23,6 +23,10 @@ app.use(express.json())
 
 app.use("/api/v1", router)
 
+app.get("/isAlive", (req, res) => {
```

Comment:
Where will it be used?

Reply:
It's used on Railway to check if a new deployment is active or not.